addons/web_editor/static/src/components/media_dialog/video_selector.js

```diff
@@ -104,7 +104,7 @@ export class VideoSelector extends Component {
             if (this.props.media) {
                 const src = this.props.media.dataset.oeExpression || this.props.media.dataset.src || (this.props.media.tagName === 'IFRAME' && this.props.media.getAttribute('src')) || '';
                 if (src) {
-                    this.state.urlInput = src;
+                    this.state.urlInput = "https:"+src;
```

Comment:
So just a linting problem here and we're good to go.
```suggestion
                    this.state.urlInput = "https:" + src;
```